Flaky test, failed to get lock on snake oil storage

The failure in:

https://github.com/equinor/komodo-releases/actions/runs/17507878019/job/49735241507

with the error:
```
self = <ert.storage.local_storage.LocalStorage object at 0x7fb14e27c8d0>
    @require_write
    def _acquire_lock(self) -> None:
        self._lock = FileLock(self.path / "storage.lock")
        try:
            self._lock.acquire(timeout=self.LOCK_TIMEOUT)
        except Timeout as e:
>           raise TimeoutError(
                f"Not able to acquire lock for: {self.path}."
                " You may already be running ERT,"
                " or another user is using the same ENSPATH."
            ) from e
E           TimeoutError: Not able to acquire lock for: /tmp/f_scout_ci/actions-runner-02/_temp/source_root/.pytest_cache/d/heat_equation_data_esmda/test_data/storage. You may already be running ERT, or another user is using the same ENSPATH.

```
indicates (possibly) that  https://github.com/equinor/ert/pull/11561 had some side effects.

--- a/tests/ert/ui_tests/cli/test_cli.py
+++ b/tests/ert/ui_tests/cli/test_cli.py
@@ -679,7 +679,7 @@ def test_that_prior_is_not_overwritten_in_ensemble_experiment(
             reals_rerun_option,
         )
 
-    with open_storage(ert_config.ens_path, mode="w") as storage:
+    with open_storage(ert_config.ens_path, mode="r") as storage:
         parameter_values = storage.get_ensemble(ensemble.id).load_parameters_numpy(
             "COEFFS", np.arange(num_realizations)
         )

--- a/tests/ert/ui_tests/cli/test_field_parameter.py
+++ b/tests/ert/ui_tests/cli/test_field_parameter.py
@@ -27,7 +27,7 @@ def test_field_param_update_using_heat_equation_enif(
     symlinked_heat_equation_storage_enif,
 ):
     config = ErtConfig.from_file("config.ert")
-    with open_storage(config.ens_path, mode="w") as storage:
+    with open_storage(config.ens_path, mode="r") as storage:
         experiment = storage.get_experiment_by_name("enif")
         [prior, posterior] = experiment.ensembles
 
@@ -174,7 +174,7 @@ def test_field_param_update_using_heat_equation_enif_snapshot(
     symlinked_heat_equation_storage_enif, snapshot, request
 ):
     config = ErtConfig.from_file("config.ert")
-    with open_storage(config.ens_path, mode="w") as storage:
+    with open_storage(config.ens_path, mode="r") as storage:
         experiment = storage.get_experiment_by_name("enif")
         prior = experiment.get_ensemble_by_name("iter-0")
         posterior = experiment.get_ensemble_by_name("iter-1")
@@ -214,7 +214,7 @@ def test_field_param_update_using_heat_equation_enif_snapshot(
 
 def test_field_param_update_using_heat_equation(symlinked_heat_equation_storage_es):
     config = ErtConfig.from_file("config.ert")
-    with open_storage(config.ens_path, mode="w") as storage:
+    with open_storage(config.ens_path, mode="r") as storage:
         experiment = storage.get_experiment_by_name("es")
         prior = experiment.get_ensemble_by_name("iter-0")
         posterior = experiment.get_ensemble_by_name("iter-1")

--- a/tests/ert/ui_tests/gui/test_main_window.py
+++ b/tests/ert/ui_tests/gui/test_main_window.py
@@ -294,7 +294,7 @@ def test_that_the_plot_window_contains_the_expected_elements(
     esmda_has_run: ErtMainWindow, qtbot
 ):
     gui = esmda_has_run
-    open_storage(gui.ert_config.ens_path, mode="w")
+    open_storage(gui.ert_config.ens_path, mode="r")
     with StorageService.init_service(
         project=os.path.abspath(gui.ert_config.ens_path),
     ):
@@ -633,7 +633,7 @@ def test_that_gui_plotter_works_when_no_data(qtbot, monkeypatch, use_tmpdir):
     ert_config = ErtConfig.from_file(config_file)
     # Open up storage to create it, so that dark storage can be mounted onto it
     # Not creating will result in dark storage hanging/lagging
-    open_storage(ert_config.ens_path, mode="w")
+    open_storage(ert_config.ens_path, mode="r")
 
     with StorageService.init_service(
         project=os.path.abspath(ert_config.ens_path),
@@ -665,7 +665,7 @@ def test_right_click_plot_button_opens_external_plotter(qtbot, use_tmpdir, monke
 
     # Open up storage to create it, so that dark storage can be mounted onto it
     # Not creating will result in dark storage hanging/lagging
-    open_storage("storage", mode="w")
+    open_storage("storage", mode="r")
 
     args_mock = Mock()
     args_mock.config = config_file

--- a/tests/ert/ui_tests/gui/test_manage_experiments_tool.py
+++ b/tests/ert/ui_tests/gui/test_manage_experiments_tool.py
@@ -393,7 +393,7 @@ ANALYSIS_SET_VAR OBSERVATIONS AUTO_SCALE POLY_OBS1_*
     config = ErtConfig.from_file("poly_localization_0.ert")
 
     notifier = ErtNotifier()
-    with open_storage(config.ens_path, mode="w") as storage:
+    with open_storage(config.ens_path, mode="r") as storage:
         notifier.set_storage(str(storage.path))
 
         tool = ManageExperimentsPanel(

--- a/tests/ert/ui_tests/gui/test_plotting_of_snake_oil.py
+++ b/tests/ert/ui_tests/gui/test_plotting_of_snake_oil.py
@@ -59,7 +59,7 @@ def plot_figure(
         args_mock.config = "config.ert"
 
     # For dark storage not to hang
-    open_storage(storage_config.ens_path, mode="w")
+    open_storage(storage_config.ens_path, mode="r")
     log_handler = GUILogHandler()
     with (
         StorageService.init_service(

--- a/tests/ert/unit_tests/storage/test_local_storage.py
+++ b/tests/ert/unit_tests/storage/test_local_storage.py
@@ -71,7 +71,7 @@ def test_create_experiment(tmp_path):
 
 def test_that_loading_non_existing_experiment_throws(tmp_path):
     with (
-        open_storage(tmp_path, mode="w") as storage,
+        open_storage(tmp_path, mode="r") as storage,
         pytest.raises(
             KeyError, match="Experiment with name 'non-existing-experiment' not found"
         ),
@@ -1132,7 +1132,7 @@ class StatefulStorageTest(RuleBasedStateMachine):
     def __init__(self) -> None:
         super().__init__()
         self.tmpdir = tempfile.mkdtemp(prefix="StatefulStorageTest")
-        self.storage = open_storage(self.tmpdir + "/storage/", "w")
+        self.storage = open_storage(self.tmpdir + "/storage/", mode="w")
         note(f"storage path is: {self.storage.path}")
         self.model: dict[UUID, Experiment] = {}
         assert list(self.storage.ensembles) == []

